app/src/main/java/com/codesoom/assignment/common/exceptions/TaskNotFoundException.java

```diff
@@ -0,0 +1,24 @@
+package com.codesoom.assignment.common.exceptions;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * 할 일을 찾을 수 없는 경우에 던집니다.
+ */
+public class TaskNotFoundException extends RuntimeException{
+
+    private Logger log = LoggerFactory.getLogger(TaskNotFoundException.class);
+
+    private String message = "해당하는 Task가 존재하지 않습니다.";
```

Comment:
다음과 같이 메시지를 작성하면 어떨까요?
```suggestion
    private String message = "존재하지 않는 Task이기 때문에 찾을 수 없습니다.";
```

- 예외의 원인이 `존재하지 않는 Task`라는 것을 알려줍니다.
- `찾을 수 없다`는 표현을 사용해 예외가 던져진 작업이 `찾기`였다는 것을 알려줍니다.